raster-foundry/raster-foundry
# Update FILE_TYPE enum to include NON_SPATIAL

## Overview

Adds a `NON_SPATIAL` value to the `FILE_TYPE` enum to support adding non-georeferenced imagery in uploads

### Checklist


### Demo

```
rasterfoundry=# update uploads set file_type = 'NON_SPATIAL';
UPDATE 0

rasterfoundry=# update uploads set file_type = 'NON_asdfasdfSPATIAL';
ERROR:  invalid input value for enum file_type: "NON_asdfasdfSPATIAL"
LINE 1: update uploads set file_type = 'NON_asdfasdfSPATIAL';
```

## Testing Instructions

- Run: `./scripts/migrate migrate`
- Open a `psql` shell and try and run the following (should not error):
`update uploads set file_type = 'NON_SPATIAL';`
- Run the following (should error):
`update uploads set file_type = 'asdfasdfasdf';`

Closes #4975
